### PR TITLE
Add new company - Pexip 

### PR DESCRIPTION
**Pull request by Niladev**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [X] I am an employee of the company mentioned and confirm all included details are correct.
- [X] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [X] The company directly hires employees. No bootcamps / freelance sites / etc
- [X] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [X] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [X] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [X] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [X] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: company-profiles/pexip.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+600+ (As of March 2022)
+
+## Remote status
+
+As a video conferencing company, enabling people to work from anywhere is what we do.
+With offices around the world, we use our own product every day to stay connected. That being said, we highly encourage socialising and give employees the opportunity to travel to the main offices when they want to be closer to their colleagues.
+
+## Region
+
+Denmark, Finland, Ireland, Switzerland, Poland, Italy, Sweden, Spain, Czech Republic, France, Netherlands, Germany, Belgium and the United Kingdom
```
Comment (dougaitken) [inline company-profiles/pexip.md:18]: Are these countries named as these are where the company can hire in, or where current employees are located?

File: company-profiles/pexip.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+-   Modern frontend technologies (React, Typescript, Angular, native mobile development)
+
+## Office locations
+
+Our main office is in Oslo, Norway but we also have offices in Gent, Belgium and Reading, England.
+
+## How to apply
+
+We'd love to see what has excited you about Pexip, and if you've got any publically available code,
+that will always help. We focus on a hiring process that gives you a proper insight into the company as well.
```
Comment (dougaitken) [inline company-profiles/pexip.md:36]: Unintentional line break?

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/pexip.md (hunk 1 above)
Comment (Niladev) [reply company-profiles/pexip.md:18]: These are the countries where we can hire from. We have employees around the world but it was meant as the places for remote.

File: company-profiles/pexip.md (hunk 2 above)
Comment (Niladev) [reply company-profiles/pexip.md:36]: Absolutely unintentional, will fix. Thanks!

**Review by dougaitken — APPROVED** [review]
I'm happy with it.

File: README.md [hunk 3]
```diff
@@ -443,6 +443,7 @@ Name | Website | Region
 [PeopleDoc](/company-profiles/peopledoc.md) | https://www.people-doc.com | Europe
 [Percona](/company-profiles/percona.md) | https://www.percona.com | Worldwide
 [Pex](/company-profiles/pex.md) | https://pex.com | Worldwide
+[Pexip](/company-profiles/pexip.md) | https://pexip.com / https://dev.pexip.com | Europe
```
Comment (dougaitken) [inline README.md:446]: Sorry @Niladev the build fails if there is more than one URL here - can you choose one and remove the other, please?

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
Stale - closing. Please open a new updated Pull Request for inclusion.

**Pull request closed without merging**